python/cugraph/cugraph/tests/link_prediction/test_overlap_mg.py

```diff
@@ -72,7 +76,10 @@ def input_expected_output(input_combo):
     input_data_path = input_combo["graph_file"]
     directed = input_combo["directed"]
     has_vertex_pair = input_combo["has_vertex_pair"]
-    G = utils.generate_cugraph_graph_from_file(input_data_path, directed=directed)
+    is_weighted = input_combo["is_weighted"]
```

Comment:
I wonder, wouldn't it be cleaner to have these column names in global variable declared at the beginning of the files?